Functions.Templates/Bindings/bindings.json

```diff
@@ -1747,6 +1747,639 @@
+                  { "value": "PrincipalId" }
+                ]
+              }
+            ]
+      },
+      {
+            "type": "GraphWebhookTrigger",
+            "direction": "trigger",
+            "displayName": "$MSGraphWebhookTrigger_displayName",
+            "documentation": "$content=Documentation\\GraphWebhookTrigger.md",
+            "filters": ["MSGraph"],
+            "enabledInTryMode": false,
+            "settings": [
+                {
+                    "name": "name",
+                    "value": "string",
+                    "defaultValue": "notification",
+                    "required": true,
+                    "label": "$MSGraphWebhookTrigger_in_name_label",
+                    "help": "$MSGraphWebhookTrigger_in_help_label"
+                },
+                {
+                    "name": "Type",
+                    "value": "enum",
+                    "defaultValue": "#Microsoft.Graph.Message",
+                    "required": true,
+                    "isHidden": true,
+                    "enum": [
+                        {
+                            "value": "#Microsoft.Graph.Message",
+                            "display": "[Outlook] Emails"
+                        },
+                        {
+                            "value": "#Microsoft.Graph.DriveItem",
+                            "display": "Drive"
+                        },
+                        {
+                            "value": "#Microsoft.Graph.Event",
+                            "display": "Calendar Events"
+                        },
+                        {
+                            "value": "#Microsoft.Graph.Contact",
+                            "display": "Contacts"
+                        }
+                    ],
+                    "label": "$MSGraphWebhookTrigger_Type_label",
+                    "help": "$MSGraphWebhookTrigger_Type_help"
+                },
+                {
+                    "name": "ChangeType",
+                    "value": "checkBoxList",
+                    "defaultValue": [
+                        "created",
+                        "updated",
+                        "deleted"
+                    ],
+                        "enum": [
+                        {
+                            "value": "created",
+                            "display": "Created"
+                        },
+                        {
+                            "value": "updated",
+                            "display": "Updated"
+                        },
+                        {
+                            "value": "deleted",
+                            "display": "Deleted"
+                        }
+                    ],
+                    "required": true,
+                    "label": "$MSGraphWebhook_in_changeType_label",
+                    "help": "$MSGraphWebhook_in_changeType_help"
+                }
+            ],
+            "rules": [
```

Comment:
If Email Messages, Contacts, or Events are selected for one input, then show all 3 checkboxes for another input (created, updated, deleted). If OneDrive is selected, only show the updated checkbox.